src/tv2-common/hotkeys/local.ts

```diff
@@ -11,15 +11,33 @@ function localSourceFullAudioHotkeyId(showStyleId: string, sourceLayer: string,
 }
 
 function localSourceFullAudioHotkeyName(source: string) {
-	return `EVS ${source}`
+	return `EVS ${source} 100%`
+}
+
+function localSourceFullAudioPick(sourceIndex: number, hotkeyType: string): number {
+	if (shouldUseCustomIndexPickerForLocalSourceAudio(hotkeyType)) {
+		return 2 * sourceIndex
+	}
+	return sourceIndex
 }
 
 function localSourceVoAudioHotkeyId(showStyleId: string, sourceLayer: string, hotkeyType: string, index: number) {
 	return `${showStyleId}_${sourceLayer}_local_vo_audio_${hotkeyType}_${index}`
 }
 
 function localSourceVoAudioHotkeyName(source: string) {
-	return `EVS ${source}`
+	return `EVS ${source} VO`
+}
+
+function localSourceVoAudioPick(sourceIndex: number, hotkeyType: string): number {
+	if (shouldUseCustomIndexPickerForLocalSourceAudio(hotkeyType)) {
+		return 1 + 2 * sourceIndex
+	}
+	return sourceIndex
+}
```

Comment:
I'm not a fan of this. Can't we use tags, and then always just pick the sourceIndex-th adlib?

Reply:
In the current setup, the same tag is used for multiple lists with the same index, which is why I did it this way.
I can have a look at how to do what you suggest.